.mocharc.yml

```diff
@@ -3,5 +3,5 @@ spec: test/**/*.test.js
 reporter: spec
 slow: 20000
 timeout: 30000
-require:
-  - isomorphic-fetch
\ No newline at end of file
+# require:
```

Comment:
Keep throwing 
```
✖ ERROR: Error: Cannot find module 'isomorphic-fetch'
```
I'm going to re-add and fix when starting to work on async calls.